.github/workflows/functional-identity.yaml

```diff
@@ -47,6 +47,7 @@ jobs:
         env:
           DEVSTACK_PATH: ${{ github.workspace }}/devstack
           ACCEPTANCE_TESTS_FILTER: "^.*identity.*$"
+          # OS_BRANCH: ${{ matrix.openstack_version }}
```

Comment:
This is intentional, to verify the test fail when OS_BRANCH isn't set.
I'll change it back after I confirmed it works as intended.